src/presenter/events-presenter.js

```diff
@@ -83,15 +81,15 @@ export default class EventsPresenter {
     this.#offers = [...this.#offersModel.offers];
     this.#destinations = [...this.#destinationsModel.destinations];
 
-    if (this.#events.length) {
+    if (!this.#events.length) {
+      render(this.#noEventsComponent, this.#eventsContainer);
+    } else {
```

Comment:
Не люблю else) Некритично но можно сделать 
```suggestion
    if (!this.#events.length) {
      render(this.#noEventsComponent, this.#eventsContainer);
      return;
    }
    
```

Reply:
Поправил